dirty_cat/_deduplicate.py

```diff
@@ -166,6 +166,35 @@ def deduplicate(
     matrix, and choosing the most frequent element in each cluster as the
     'correct' spelling. This method works best if the true number of
     categories is significantly smaller than the number of observed spellings.
+
+    Examples
+    --------
+    >>> from dirty_cat.datasets import make_deduplication_data
+    >>> duplicated = make_deduplication_data(examples=['black', 'white'],
+                                             entries_per_example=[5, 5],
+                                             prob_mistake_per_letter=0.3,
+                                             random_state=42)
+
+    >>> duplicated
+    ['blacn', 'black', 'black', 'black', 'black',
+     'hvite', 'white', 'white', 'white', 'white']
+
+    To deduplicate the data:
+
+    >>> deduplicated = deduplicate(duplicated)
```

Comment:
```suggestion
    >>> deduplicate_correspondence = deduplicate(duplicated)
```
If I'm not wrong, this variable is not the deduplicated version of the `duplicated` table. It's of a different nature, so the name should reflect this.